akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -154,10 +155,16 @@ private[http] object HttpServerBluePrint {
 
     val responseRendererFactory = new HttpResponseRendererFactory(serverHeader, responseHeaderSizeHint, log)
 
+    val errorHandler: Throwable ⇒ Unit = {
+      // socket timeouts should not result in errors in the log. See 19058.
```

Comment:
It's not really "socket timeout", its "idle timeout" - socket can be perfectly fine, just nothing happening on it.